### PR TITLE
1164 fixing test details page UI issues

### PR DESCRIPTION
**Pull request by xoscar**
This PR fixes the multiple UI issues described as part of this ticket https://github.com/kubeshop/tracetest/issues/1164

## Changes

- Fixes the mode tabs
- Updates the DAG controls the position
- Changes the response headers row's design

## Fixes

- https://github.com/kubeshop/tracetest/issues/1164

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

Screen Shot 2022-09-13 at 12 01 32
Screen Shot 2022-09-13 at 12 01 37
Screen Shot 2022-09-13 at 12 01 58

### CODE REVIEW

File: web/src/components/HeaderRow/HeaderRow.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import {THeader} from 'types/Test.types';
+import Highlighted from '../Highlighted';
+import * as S from './HeaderRow.styled';
+
+interface IProps {
+  header: THeader;
+  onCopy(value: string): void;
+}
+
+const HeaderRow = ({header: {key = '', value = ''}, onCopy}: IProps) => {
```
Comment (xoscar) [inline web/src/components/HeaderRow/HeaderRow.tsx:13]: New header row component

File: web/src/components/RunDetailLayout/RunDetailLayout.styled.ts [hunk 2]
```diff
@@ -34,11 +34,27 @@ export const ContainerHeader = styled.div`
...
+
   .ant-tabs-tab {
-    border: ${({theme}) => `1px solid ${theme.color.borderLight}`};
     font-weight: 600;
     padding: 5px 16px;
     margin: 7px 0;
+    border: ${({theme}) => `1px solid ${theme.color.borderLight}`};
+    border-right: none;
+
+    &:nth-of-type(1) {
```
Comment (xoscar) [inline web/src/components/RunDetailLayout/RunDetailLayout.styled.ts:48]: Styles for the first and last tab

**Review by jorgeepc — APPROVED** [review]
🔥

**Review by cescoferraro — APPROVED** [review]
Thks @xoscar

**Pull request merged**